routes/articles.go

```diff
@@ -56,11 +52,23 @@ func verifyArticle(article  schema.Article) (schema.Article, error) {
 }
 
 // TODO: better error handling/DB constraints (duplicates, missing fields, etc.)
-func addArticleToDB(article schema.Article) (int64, error) {
+func addArticleToDB(article schema.Article , c *gin.Context) (int64, error) {
     conn := db.GetDB()
-
     newArticle, err := verifyArticle(article)
-    newArticle.UserID = 1
+
```

Comment:
more than one empty line between lines is yucky. i would prefer something more like below with lines 1, 2, and 3. 4 is incorrectly spaced here.

```
{
  line of code 1
  line of code 2

  line of code 3


  line of code 4 that is too far from 3
}
```